certora/harness/MinterGatewayHarness.sol

```diff
@@ -0,0 +1,46 @@
+// SPDX-License-Identifier: GPL-3.0
+
+pragma solidity 0.8.23;
+import "../../src/MinterGateway.sol";
+
+contract MinterGatewayHarness is MinterGateway {
+    constructor(address ttgRegistrar_, address mToken_) MinterGateway(ttgRegistrar_, mToken_) {}
+
+    function totalMSupply() public view returns (uint240) {
+        // Note this is also used to determine the total M supply in
+        // functions implemented in MinterGatway
+        return uint240(IMToken(mToken).totalSupply()); 
+    }
+
+    /******************************************************************************************************************\
+    |                                                     Getters                                                      |
+    \******************************************************************************************************************/
+
+    function getLatestIndexInMinterGateway() public view returns (uint128) {
+        return latestIndex;
+    }
+
+    function getLatestRateInMinterGateway() public view returns (uint32) {
+        return _latestRate;
+    }
+
+    function getLatestUpdateTimestampInMinterGateway() public view returns (uint40) {
+        return latestUpdateTimestamp;
+    }
```

Comment:
Why is this needed, isn't `latestUpdateTimestamp` already a public getter? And why is there a redundant `InMinterGateway` suffix in the function name?